ansible/roles/util-pgpass/tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- name: place .pgpass
+  sudo: yes
+  template: src=postgres.pgpass.j2 dest=~/.pgpass mode=0600 backup=yes
```

Comment:
I made the destination `~root/.pgpass`. It's still explicit and it will work in the (unlikely) case where root's home directory is something other than `/root`.